gradio/blocks.py

```diff
@@ -463,6 +463,7 @@ def __init__(
         self.share_url = None
         self.width = None
         self.height = None
+        self.api_open = False
```

Comment:
My feeling is that the API should be open by default to preserve the current behavior.